logger.go

```diff
@@ -26,10 +26,9 @@ func NewLogger() *Logger {
 
 func (l *Logger) ServeHTTP(rw http.ResponseWriter, r *http.Request, next http.HandlerFunc) {
 	start := time.Now()
-	l.Printf("Started %s %s", r.Method, r.URL.Path)
 
 	next(rw, r)
 
 	res := rw.(ResponseWriter)
-	l.Printf("Completed %v %s in %v", res.Status(), http.StatusText(res.Status()), time.Since(start))
+	l.Printf("%v | %v | \t %v | %v | %v %v", start.Format(time.RFC3339), res.Status(), time.Since(start), r.Host, r.Method, r.URL.Path)
```

Comment:
@yageek I agree that [making this string with a template is more expensive than a format string](https://gist.github.com/meatballhat/cd929f4877fb013a4fd8c5bc9b8f1031#file-results-txt).  I also think of the default negroni logging middleware as a beginner's helper, or a reasonable default for most web apps, rather than a production-critical-zomg-tuned logger.  I want to avoid future patches that propose new format strings if possible, which I think can happen if we allow users of negroni to set the string to whatever they like.  I prefer a template string over a `fmt` string because then we aren't dictating argument order, and (hopefully) the strings are seen as more self-explanatory.